tests/rbac_test.go

```diff
@@ -0,0 +1,183 @@
+
+
+				By("Login with the user kubevirt-test-user")
+				args = []string{"login", "-u", ktests.UsernameTestUser, "-p", "123456"  }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Create a VM with yaml")
+				args = []string{"apply", "-f", "tests/manifests/vm-cirros.yaml"}
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Login with the user test_user")
+				args = []string{"login", "-u", "test_admin", "-p", "123456"  }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+
+				By("Remove admin role to user kubevirt-test-user")
+				args = []string{"adm", "policy", "remove-role-from-user", "edit", ktests.UsernameTestUser, "-n", tests.NamespaceTestDefault }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Delete identity of user kubevirt-test-user")
+				args = []string{"delete", "identity", "allow_all_auth:kubevirt-test-user" }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+		})
+
+		It("with view permission", func ()  {
+				By("Add admin role to user kubevirt-test-user")
+				args := []string{"adm", "policy", "add-role-to-user", "view", ktests.UsernameTestUser, "-n", tests.NamespaceTestDefault }
+				_, err := tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+
+				By("Login with the user kubevirt-test-user")
+				args = []string{"login", "-u", ktests.UsernameTestUser, "-p", "123456"  }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Create a VM with yaml")
+				args = []string{"apply", "-f", "tests/manifests/vm-cirros.yaml"}
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).To(HaveOccurred())
+
+
+				By("Login with the user test_user")
+				args = []string{"login", "-u", "test_admin", "-p", "123456"  }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+
+				By("Remove admin role to user kubevirt-test-user")
+				args = []string{"adm", "policy", "remove-role-from-user", "view", ktests.UsernameTestUser, "-n", tests.NamespaceTestDefault }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Delete identity of user kubevirt-test-user")
+				args = []string{"delete", "identity", "allow_all_auth:kubevirt-test-user" }
+				_, err = tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+		})
+
+		Describe("Without permissions", func() {
+
+			It("should not be able to access subresource endpoint", func() {
+
+				By("Login with the user kubevirt-test-user")
+				args := []string{"login", "-u", ktests.UsernameTestUser, "-p", "123456"  }
+				_, err := tests.RunCommand("oc", args...)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Create a VM with yaml")
+				args = []string{"apply", "-f", "tests/manifests/vm-cirros.yaml"}
```

Comment:
this is only verifying that the user can not create a VMI, It does not guarantee they don't have the permissions to view the VMI's console.